webapp/src/components/assetEditor/assetSidebar.tsx

```diff
@@ -119,7 +119,7 @@ class AssetSidebarImpl extends React.Component<AssetSidebarProps, AssetSidebarSt
         const { asset, isGalleryAsset } = this.props;
         const { showDeleteModal } = this.state;
         const details = this.getAssetDetails();
-        const name = asset?.meta?.displayName || lf("No asset selected");
+        const name = asset ? asset.meta?.displayName || asset.id || lf("Unnamed") : lf("No asset selected");
```

Comment:
Instead of asset.id, maybe pxt.getShortIDForAsset(asset)? It removes the namespace